src/styles/accountLoginForm.styl

```diff
@@ -1,6 +1,10 @@
 @import cozy-ui
 
 :local // @stylint ignore
+    .account-form-login
```

Comment:
We can avoid using CSS Modules if we just need a class. We only have to specify an explicit class in JSX, for example for this line https://github.com/cozy/cozy-collect/blob/master/src/components/AccountLoginForm.jsx#L15 : 
```jsx
<div className='account-form-login'>
```